pdf-generator/POSTINSTALL.md

```diff
@@ -16,24 +17,26 @@ Update your Firebase Storage security rules to allow reads from template path:
 If `OUTPUT_STORAGE_BUCKET` is set, you need to allow writes to where generated pdf files are going to be stored too:
 
     match /b/${param:OUTPUT_STORAGE_BUCKET}/o {
-      // Allow access to the path where generated pdf files will be stored
+      // Allow access to the generated pdf file path
       match /{outputs_path} {
         allow write;
       }
     }
 
 Check [here](https://firebase.google.com/docs/storage/security) for more information about security rules.
 
-***
+---
 
 ## Executing the function
 
 You can start testing this extension right away.
 
-1.  Create a template based on this [guide](https://github.com/sassanh/template-to-pdf/blob/main/pdf-generator/PREINSTALL.md). Make note of the Handlebars.js parameters you've used in the template, as they should match the request you create in the next step.
```

Comment:
Remark automatically changes 1, 1, 1 sequence to 1, 2, 3. (I prefer 1, 1, 1)
Which formatter are you using? I will try to use the same thing.

Reply:
Ah ok. We are actively improving our guidelines and documentation.

I've been using [markdown](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) in VsCode.

Open to any suggestions and recommendations if this improved the documentation standard.

Reply:
Thanks! I think the documentation standard is fine, I just needed to setup my formatter.